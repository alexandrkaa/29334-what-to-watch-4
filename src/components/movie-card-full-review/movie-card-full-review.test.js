import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardFullReview from './movie-card-full-review.jsx';

const comment = {
  movieId: 1,
  id: 1000,
  author: `John Doe`,
  text: `Those an equal point no years do. Depend warmth fat but her but played. Shy and subjects wondered trifling pleasant. Prudent cordial comfort do no on colonel as assured chicken. Smart mrs day which begin. Snug do sold mr it if such. Terminated uncommonly at at estimating. Man behaviour met moonlight extremity acuteness direction.`,
  date: `01.01.2000`,
  rating: 3,
};

it(`<MovieCardFullReview /> should match snapshot`, () => {
  const tree = renderer
    .create(
        <MovieCardFullReview
          comment={comment}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
