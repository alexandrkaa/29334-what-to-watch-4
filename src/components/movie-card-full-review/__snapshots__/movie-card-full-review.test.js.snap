// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardFullReview /> should match snapshot 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Those an equal point no years do. Depend warmth fat but her but played. Shy and subjects wondered trifling pleasant. Prudent cordial comfort do no on colonel as assured chicken. Smart mrs day which begin. Snug do sold mr it if such. Terminated uncommonly at at estimating. Man behaviour met moonlight extremity acuteness direction.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        John Doe
      </cite>
      <time
        className="review__date"
        dateTime="01.01.2000"
      >
        01.01.2000
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    3
  </div>
</div>
`;
